### PR TITLE
RPP Test Suite - Remove Empty Lines and Cleanup

### PR DESCRIPTION
**Pull request by HazarathKumarM**
- Clean up test suite  code and removes empty lines

### CODE REVIEW

**Comment by HazarathKumarM** [comment]
@r-abishek 
I am attaching both the old log logs and new logs after the changes for the image test suite
[old_unittest_log.txt](https://github.com/user-attachments/files/16474184/old_unittest_log.txt)
[old_qa_log.txt](https://github.com/user-attachments/files/16474185/old_qa_log.txt)
[old_perf_log.txt](https://github.com/user-attachments/files/16474186/old_perf_log.txt)
[new_unittest_log.txt](https://github.com/user-attachments/files/16474187/new_unittest_log.txt)
[new_perf_log.txt](https://github.com/user-attachments/files/16474188/new_perf_log.txt)
[new_qa_log.txt](https://github.com/user-attachments/files/16474189/new_qa_log.txt)

File: utilities/test_suite/HIP/Tensor_hip.cpp [hunk 1]
```diff
@@ -1456,7 +1456,13 @@ int main(int argc, char **argv)
 
         if (testType == 0)
         {
-            cout << "\n\nGPU Backend Wall Time: " << wallTime <<" ms/batch"<< endl;
+            if(!qaFlag)
```
Comment (sampath1117) [inline utilities/test_suite/HIP/Tensor_hip.cpp:1459]: You can restructure to something like below?
Also add a comment explaining the condition check
```
if (testType == 0)
{
    cout <<"\n\n";
    if(!qaFlag && noOfIterations > 1)
        cout <<"Execution Timings for Iteration "<< iterCount + 1 <<":"<<endl;
    cout << "GPU Backend Wall Time: " << wallTime <<" ms/batch";
```

File: utilities/test_suite/HIP/runAudioTests.py [hunk 2]
```diff
@@ -73,7 +73,7 @@ def generate_performance_reports(RESULTS_DIR):
     print(dfPrint_noIndices)
 
 def run_unit_test_cmd(srcPath, case, numRuns, testType, batchSize, outFilePath):
-    print("./Tensor_audio_hip " + srcPath + " " + str(case) + " " + str(numRuns) + " " + str(testType) + " " + str(numRuns) + " " + str(batchSize))
+    print("\n./Tensor_audio_hip " + srcPath + " " + str(case) + " " + str(numRuns) + " " + str(testType) + " " + str(numRuns) + " " + str(batchSize))
```
Comment (sampath1117) [inline utilities/test_suite/HIP/runAudioTests.py:76]: Why this is added?
if there is a requirement
can't we move this outside this print statement?

File: utilities/test_suite/HIP/runAudioTests.py [hunk 3]
```diff
@@ -237,6 +234,7 @@ def rpp_test_suite_parser_and_validator():
             srcPath = inFilePath
 
     if case not in supportedCaseList:
+        print("\nCase " + case + " is not supported.")
```
Comment (sampath1117) [inline utilities/test_suite/HIP/runAudioTests.py:237]: Adding this print statement here again, prints a lot of statements for which cases are not supported
We need to add a print statement **only if the none of the cases requested by user is supported** 

if noCaseSupported:
 print "case numbers {case_list} is not supported"

File: utilities/test_suite/HIP/runMiscTests.py [hunk 4]
```diff
@@ -97,15 +97,13 @@ def run_performance_test_with_profiler_cmd(loggingFolder, numDims, case, numRuns
...
 def run_test(loggingFolder, numDims, case, numRuns, testType, toggle, batchSize, outFilePath, additionalArg, profilingOption = 'NO'):
-    print("\n\n\n\n")
-    print("--------------------------------")
-    print("Running a New Functionality...")
-    print("--------------------------------")
     if testType == 0:
         run_unit_test_cmd(numDims, case, numRuns, testType, toggle, batchSize, outFilePath, additionalArg)
     elif testType == 1 and profilingOption == "NO":
+        print("\n")
         run_performance_test_cmd(loggingFolder, numDims, case, numRuns, testType, toggle, batchSize, outFilePath, additionalArg)
```
Comment (sampath1117) [inline utilities/test_suite/HIP/runMiscTests.py:104]: we can add print("\n") before all the if conditions in this run_test and remove the "\n" added in  run_unit_test_cmd

File: utilities/test_suite/common.py [hunk 5]
```diff
@@ -310,15 +310,15 @@ def print_performance_tests_summary(logFile, functionalityGroupList, numRuns):
...
-    print("\n\nFunctionality\t\t\t\t\t\tFrames Count\tmax(ms/batch)\t\tmin(ms/batch)\t\tavg(ms/batch)\n")
+    header_format = "{:<70} {:<15} {:<15} {:<15} {:<15}"
+    row_format = "{:<70} {:<15} {:<15} {:<15} {:<15}"
+    print("\n" + header_format.format("Functionality", "Frames Count", "max(ms/batch)", "min(ms/batch)", "avg(ms/batch)") + "\n")
     if len(functions) != 0:
-        maxCharLength = len(max(functions, key = len))
-        functions = [x + (' ' * (maxCharLength - len(x))) for x in functions]
         for i, func in enumerate(functions):
-            print(func + "\t" + str(frames[i]) + "\t\t" + str(maxVals[i]) + "\t" + str(minVals[i]) + "\t" + str(avgVals[i]))
+            print(row_format.format(func, str(frames[i]), str(maxVals[i]), str(minVals[i]), str(avgVals[i])))
```
Comment (sampath1117) [inline utilities/test_suite/common.py:321]: Please change variable name from snake case  to camel case
row_format to rowFormat
header_format to headerFormat

**Review by sampath1117 — COMMENTED** [review]
@HazarathKumarM 
Please address minor comments

File: utilities/test_suite/HIP/Tensor_hip.cpp (hunk 1 above)
Comment (HazarathKumarM) [reply utilities/test_suite/HIP/Tensor_hip.cpp]: Done

File: utilities/test_suite/HIP/runAudioTests.py (hunk 2 above)
Comment (HazarathKumarM) [reply utilities/test_suite/HIP/runAudioTests.py:76]: This was added for an empty line before printing the contents. Without it, the output does not look good.

File: utilities/test_suite/HIP/runAudioTests.py (hunk 3 above)
Comment (HazarathKumarM) [reply utilities/test_suite/HIP/runAudioTests.py]: Done

File: utilities/test_suite/common.py (hunk 5 above)
Comment (HazarathKumarM) [reply utilities/test_suite/common.py:321]: Done

File: utilities/test_suite/HIP/Tensor_hip.cpp [hunk 6]
```diff
@@ -1456,7 +1456,10 @@ int main(int argc, char **argv)
 
         if (testType == 0)
         {
-            cout << "\n\nGPU Backend Wall Time: " << wallTime <<" ms/batch"<< endl;
+            cout <<"\n\n";
+            if(noOfIterations > 1)
+                cout <<"Execution Timings for Iteration "<< iterCount+1 <<":"<<endl;
```
Comment (r-abishek) [inline utilities/test_suite/HIP/Tensor_hip.cpp:1461]: Ensure consistent spacing before and after << and arithmetic/logical operators

**Review by r-abishek — APPROVED** [review]

**Pull request merged**